### ISSUE TITLE
gulp issue on OSX

### ISSUE THREAD

**Issue by dpastoor** [airtoxin/Electron-React-Boilerplate#2]
```
[17:13:22] gulp-atom-shell unzip darwin-x64 v0.20.6 atom-shell.

/Users/devin/Repos/Electron-React-Boilerplate/node_modules/gulp-atom/lib/index.js:204
      throw new util.PluginError(PLUGIN_NAME, results.stderr || 'unknow error
            ^
Error in plugin 'gulp-atom-shell'
unknow error , maybe you can try delete the zip packages.
    at ChildProcess.<anonymous> (/Users/devin/Repos/Electron-React-Boilerplate/node_modules/gulp-atom/lib/index.js:204:13)
    at ChildProcess.emit (events.js:98:17)
    at Process.ChildProcess._handle.onexit (child_process.js:820:12)

npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.18
npm ERR! code ELIFECYCLE
npm ERR! atomshell-react-boilerplate@0.0.1 start: `gulp`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the atomshell-react-boilerplate@0.0.1 start script 'gulp'.
npm ERR! This is most likely a problem with the atomshell-react-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get their info via:
npm ERR!     npm owner ls atomshell-react-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/devin/Repos/Electron-React-Boilerplate/npm-debug.log
```

I tried to manually reinstall gulp and gulp-atom-shell but no luck so far.

**Comment by airtoxin** [comment]
@dpastoor thanks for issue.
I fixed 653ffb602673d41e6165da502657345f13f14b9d.

**Comment by dpastoor** [comment]
closing, as it works on osx as mentioned in #3
